CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-rc.1 December 2, 2021
+
+## Bug fixes 🐞
+* Fix `replaceWith` template to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
+* Do not allow to use one `associatedFeatureId` with multiple view annotations. ([#896](https://github.com/mapbox/mapbox-maps-android/pull/896))
+* Fix an issue where shove gesture was not detected when angle between touch points are not horizontal. ([#875](https://github.com/mapbox/mapbox-maps-android/pull/875))
+* Fix gestures at high camera pitch near horizon line. ([#927](https://github.com/mapbox/mapbox-maps-android/pull/927), [#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
+* Fix fading tiles layout visibility issue on globe view projection zooming in/out. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
+* Add config to enable simultaneous zoom and rotate gesture. ([#885](https://github.com/mapbox/mapbox-maps-android/pull/885))
```

Comment:
shall we mention the default behaviour is changed(fixed) as we now by default allow simultaneous zoom and rotate gesture?

Reply:
```suggestion
* Allow simultaneous zoom and rotate gesture by default and add `GesturesSettings#simultaneousRotateAndPinchToZoomEnabled` config option. ([#885](https://github.com/mapbox/mapbox-maps-android/pull/885))
```

Reply:
should we mention it in changelog or in release note?

Reply:
I think we should